tests/sys/sys_build/Cargo.toml

```diff
@@ -2,13 +2,9 @@
 name = "sys_build"
 version = "0.0.1"
 
-[lib]
-name = "sys_build"
-
 [features]
-full = [
-	"secret-sys",
-]
+full = ["secret-sys"]
+"3.16" = ["gtk-sys/3.16"]
```

Comment:
Now that gir understand relative directories in config files the script is more of a liability. It can be replaced with two commands essentially. I had to do that in appveyor because cargo tended to rebuild gir when run from the script.